streams-kafka/src/main/java/io/kipe/streams/kafka/processors/TableBuilder.java

```diff
@@ -62,6 +62,13 @@
  * deserialization is defined using the keySerde and valueSerde objects. The TableBuilder is then instantiated with the
  * streamsBuilder, stream, keySerde and valueSerde as arguments, and the build method is called to create the topology
  * component that groups incoming data by key and stores it in a state store accessible through the transformer.
+ * <p>
+ * The table below shows the table command with its stateful and internal topics details:
+ * <pre>
+ * | command | stateful | internal topics                               |
+ * |---------|----------|-----------------------------------------------|
+ * | table   | yes      | {topicsBaseName}-tablebuilder-processor-store |
```

Comment:
We should rename it to `{topicsBaseName}-table-processor-store` to follow the same naming structure for every internal topic. Please do it as part of this PR.